src/Components/Publishing/Icon/IconShareArrow.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from "react"
+
+export class IconShareArrow extends Component<any> {
+  render() {
+    return (
+      <svg
+        width="12px"
+        height="12px"
+        viewBox="0 0 12 12"
+        version="1.1"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <defs>
+          <polygon
+            id="path-1"
```

Comment:
I think whatever SVG exporter design uses appends these `id` attributes, but we should strip them from the SVG because realistically we will have multiples of this SVG on an infinite scroll page and `id`'s should technically only exist once on a page.